tests/test_with_dummy.py

```diff
@@ -425,6 +445,38 @@ def test_recursive_lazy_autogen():
         )
         mkinit_rec_lazy_autogen.a_very_nested_function()
 
+def test_typed_pyi_file():
+    """
+    xdoctest ~/code/mkinit/tests/test_with_dummy.py test_recursive_lazy_autogen
+    """
+    import mkinit
+    import os
+
+    if sys.version_info[0:2] < (3, 7):
+        pytest.skip('Only 3.7+ has lazy imports')
+
+    cache_dpath = ub.Path.appdir("mkinit/tests").ensuredir()
+    paths = make_dummy_package(cache_dpath)
+    pkg_path = paths["root"]
+
+    mkinit.autogen_init(
+        pkg_path,
+        options={"lazy_loader": 1, "lazy_loader_typed": True, "relative": True},
```

Comment:
If relative is not `True`, mkinit runs but attempts at importing the module fail. The CLI currently (silently) sets relative to True whenever `--lazy_loader_typed` is specified, so this shouldn't come up for users, but perhaps you want some extra validation? Not sure where the best place to put it is though